Orange/widgets/settings.py

```diff
@@ -791,22 +790,25 @@ def close_context(self, widget):
         self.settings_from_widget(widget)
         widget.current_context = None
 
-    def settings_to_widget(self, widget, *args):
+    def settings_to_widget(self, widget, *args, decode_setting=None):
```

Comment:
I find this barely better than patching.

Reply:
but it is more functional :)